doc/configcommands.dsv

```diff
@@ -12,6 +12,7 @@ browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use whe
 cache-file||<path>||"~/.newsboat/cache.db" or "~/.local/share/cache.db" (see "Files" section)||This configuration option sets the cache file. This is especially useful if the filesystem of your home directory doesn't support proper locking (e.g. NFS).||cache-file "/tmp/testcache.db"
 cleanup-on-quit||[yes/no]||yes||If set to `yes`, then the cache gets locked and superfluous feeds and items are removed, such as feeds that can't be found in the urls configuration file anymore.||cleanup-on-quit no
 color||<element> <fgcolor> <bgcolor> [<attribute> ...]||n/a||Set the foreground color, background color and optional attributes for a certain element.||color background white black
+confirm-markAllFeedsRead||[yes/no]||yes||If set to `yes`, then newsboat will ask for confirmation whether the user wants to mark all feeds as read.||confirm-markAllFeedsRead no
```

Comment:
Please replace camelCase to be in line with other settings: `confirm-mark-all-feeds-read`.